Makefile

```diff
@@ -439,7 +439,8 @@ run-server: start-docker ## Starts the server.
 	@echo Running mattermost for development
 
 	mkdir -p $(BUILD_WEBAPP_DIR)/dist/files
-	$(GO) run $(GOFLAGS) $(GO_LINKER_FLAGS) $(PLATFORM_FILES) --disableconfigwatch &
+	$(GO) run $(GOFLAGS) $(GO_LINKER_FLAGS) $(PLATFORM_FILES) --disableconfigwatch | \
```

Comment:
This is great, @cpoile! Just to confirm, can you test the experience when "things go wrong" -- e.g. if your code fails to compile. Is any of that output obscured?

Reply:
@lieut-data It doesn't look like output is obscured -- I tested it with a broken source file and since the output piped into logrus isn't json, it doesn't find any fields and just passes on the string. To see why, see: human.ProcessLogs, ParseLogMessage, and parseLogMessage in the logrus pkg.

Reply:
Perfect!